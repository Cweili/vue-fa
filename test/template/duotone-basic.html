<p>
  <fa :icon="fadFlag"/> Flag
</p>
