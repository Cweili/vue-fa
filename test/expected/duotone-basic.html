<p data-server-rendered="true"><svg viewBox="0 0 512 512" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg" style="height:1em;overflow:visible;vertical-align:-.125em;"><g transform="translate(256 256)"><g transform=""><path d="M512 91.33v277c0 11.31-7.1 21.88-18.5 26.47C317.7 465 281.7 331.25 96 416V102a56.57 56.57 0 0 0 14.64-15c194.19-74.48 184.75 58.25 352-20.08C485.2 56.31 512 68.26 512 91.33z" fill="currentColor" fill-opacity="0.4" transform="translate(-256 -256)"></path> <path d="M120 56a55.93 55.93 0 0 1-24 46v388a22 22 0 0 1-22 22H54a22 22 0 0 1-22-22V102a56 56 0 1 1 88-46z" fill="currentColor" fill-opacity="1" transform="translate(-256 -256)"></path></g></g></svg> Flag
</p>